Components/DayCalendar.js

```diff
@@ -9,50 +12,251 @@ var {
   ScrollView,
   View,
   Text,
+  PanResponder,
+  Image,
 } = React;
 
+type Event = {
+  title: string;
+  startAt: Date;
+  length: number;
+};
+
 var HOURS_COUNT = 25;
 var HOURS = [];
 for (var i = 0; i < HOURS_COUNT; i++) {
-    var hour = moment().hour(i).second(0).minute(0).format('h:mma');
+    var hour = moment().hour(i).minute(0).second(0).format('h:mma');
```

Comment:
I find a way to declare block-scoped variable. In ES6 I can use `let` or `const`. https://babeljs.io/docs/learn-es6/#let-const